README.md

```diff
@@ -2,7 +2,7 @@
 
 ## <a name="about"></a> About node-oracledb
 
-The node-oracledb add-on for Node.js powers high performance Oracle
+The node-oracledb module for Node.js powers high performance Oracle
```

Comment:
Technically its an add-on (or addon, depending on which English variant etc etc you like), see https://nodejs.org/api/addons.html